koku/masu/processor/tasks.py

```diff
@@ -873,6 +873,9 @@ def update_cost_model_costs(
         None
 
     """
+    # Override cost model start date to calculate costs for full month
+    LOG.info("overriding cost model start date to process full month")
+    start_date = DateHelper().month_start(start_date)
```

Comment:
So, I think with this approach we loose the ability to update cost model costs for previous months through our masu endpoint.

Maybe we can check to see if the date is within the current month and overwrite it to this month start?

Reply:
Ignore me, I can't read. `month_start` != `this_month_start`